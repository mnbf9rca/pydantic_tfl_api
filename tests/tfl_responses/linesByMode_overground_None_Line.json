{"status_code": 200, "headers": {"Date": "Mon, 15 Jul 2024 15:02:47 GMT", "Content-Type": "application/json; charset=utf-8", "Content-Length": "274", "Connection": "keep-alive", "Cache-Control": "public, must-revalidate, max-age=43200, s-maxage=86400", "Via": "1.1 varnish", "Content-Encoding": "gzip", "Accept-Ranges": "bytes", "Age": "12142", "Access-Control-Allow-Headers": "Content-Type", "Access-Control-Allow-Methods": "GET,POST,PUT,DELETE,OPTIONS", "Access-Control-Allow-Origin": "*", "Api-Entity-Payload": "Line", "X-Backend": "api", "X-Cache": "HIT", "X-Cache-Hits": "2", "X-Cacheable": "Yes. Cacheable", "X-Frame-Options": "deny", "X-Proxy-Connection": "unset", "X-TTL": "86400.000", "X-TTL-RULE": "0", "X-Varnish": "2026252039 2023948131", "X-AspNet-Version": "4.0.30319", "X-Operation": "Line_GetByModeByPathModes", "X-API": "Line", "CF-Cache-Status": "DYNAMIC", "Strict-Transport-Security": "max-age=31536000", "Set-Cookie": "_cfuvid=mqG89o.MjcC9.2TRZL0uVfQEESFnPOZedhz8f1_OOpQ-1721055767027-0.0.1.1-604800000; path=/; domain=.tfl.gov.uk; HttpOnly; Secure; SameSite=None", "Server": "cloudflare", "CF-RAY": "8a3aa2af99ce63f0-LHR"}, "url": "https://api.tfl.gov.uk/Line/Mode/overground", "content": "[{\"$type\":\"Tfl.Api.Presentation.Entities.Line, Tfl.Api.Presentation.Entities\",\"id\":\"london-overground\",\"name\":\"London Overground\",\"modeName\":\"overground\",\"disruptions\":[],\"created\":\"2024-07-09T13:08:01.777Z\",\"modified\":\"2024-07-09T13:08:01.777Z\",\"lineStatuses\":[],\"routeSections\":[],\"serviceTypes\":[{\"$type\":\"Tfl.Api.Presentation.Entities.LineServiceTypeInfo, Tfl.Api.Presentation.Entities\",\"name\":\"Regular\",\"uri\":\"/Line/Route?ids=London Overground&serviceTypes=Regular\"},{\"$type\":\"Tfl.Api.Presentation.Entities.LineServiceTypeInfo, Tfl.Api.Presentation.Entities\",\"name\":\"Night\",\"uri\":\"/Line/Route?ids=London Overground&serviceTypes=Night\"}],\"crowding\":{\"$type\":\"Tfl.Api.Presentation.Entities.Crowding, Tfl.Api.Presentation.Entities\"}}]"}