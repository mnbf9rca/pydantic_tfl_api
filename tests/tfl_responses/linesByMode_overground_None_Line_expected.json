["{\"id\":\"london-overground\",\"name\":\"London Overground\",\"mode_name\":\"overground\",\"disruptions\":[],\"created\":\"2024-07-12T10:26:26.827000Z\",\"modified\":\"2024-07-12T10:26:26.827000Z\",\"line_statuses\":[],\"route_sections\":[],\"service_types\":[{\"name\":\"Regular\",\"uri\":\"/Line/Route?ids=London Overground&serviceTypes=Regular\"},{\"name\":\"Night\",\"uri\":\"/Line/Route?ids=London Overground&serviceTypes=Night\"}],\"crowding\":{\"passenger_flows\":null,\"train_loadings\":null},\"content_expires\":\"2024-07-16T03:40:43Z\",\"shared_expires\":\"2024-07-16T15:40:43Z\"}"]